test/Garnet.test/RespCustomCommandTests.cs

```diff
@@ -526,7 +526,7 @@ public void CreateTestLibrary(string[] namespaces, string[] referenceFiles, stri
                 var source = File.ReadAllText(filePath);
                 var stringText = SourceText.From(source, Encoding.UTF8);
                 return SyntaxFactory.ParseSyntaxTree(stringText,
-                    CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.CSharp10), string.Empty);
+                    CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.Latest), string.Empty);
```

Comment:
The test failure was caused by https://github.com/microsoft/garnet/pull/201/commits/a7d416eaf180ba34f12a74119f8e5d4be02a16b3 where I updated ``ReadWriteTxn`` to use new C# 12 features. This caused `RespCustomCommandTests.CreateTestLibraries` to fail as it pulls the ``ReadWriteTxn.cs`` in for compilation:
https://github.com/microsoft/garnet/blob/4971e20e884da234cfcbcec6e59b939b54a0fa06/test/Garnet.test/RespCustomCommandTests.cs#L610-L614